rockcraft.yaml

```diff
@@ -1,32 +1,40 @@
 name: snapcraft-core24
 base: ubuntu@24.04
-version: 8.4.1
-summary: easily create core24 snaps
+version: 8.9.2
+summary: Package, distribute, and update any app for Linux and IoT.
 description: |
-  Snapcraft aims to make upstream developers' lives easier and as such is not
-  a single toolset, but instead is a collection of tools that enable the
-  natural workflow of an upstream to be extended with a simple release step
-  into Snappy.
-  This version of Snapcraft is able to build core24 snaps in destructive mode
-  only.
+  Snapcraft is the command-line build tool for packaging and distributing software and apps in the snap container format.
+
+  The tool packages apps across many supported languages, build tools, and frameworks, such as Python, C and C++, Rust, Node, and GNOME. Snaps can be tested, debugged, and locally shared before being published to the global Snap Store and private stores. It uses simple commands to manage and monitor releases at a granular level.
+
+  It solves the problems of dependency management and architecture support by bundling all of a software’s libraries into the container itself, and provides a way to package any app, program, toolkit, or library for all major Linux distributions and IoT devices.
+
+  Snapcraft is for developers, package maintainers, fleet administrators, and hobbyists who are interested in publishing snaps for Linux and IoT devices.
 license: GPL-3.0
 platforms:
   amd64: #TODO
 
 parts:
   snapcraft:
+    # We can't use the uv plugin because it rewrites shebangs in scripts from the python3-venv package.
+    # For example, the uv plugin changes the shebang in `/usr/bin/dpkg-maintscript-helper` from dpkg
+    #  from `#! /bin/sh` to `#! /bin/python3`. This causes Snapcraft to fail when installing build packages.
```

Comment:
Is this considered a bug in rockcraft?

Reply:
I see, I think we're saying the same thing.

https://github.com/canonical/rockcraft/issues/889